### ISSUE TITLE
Filebeat 1.0-beta4 preview - documentation feedback

### ISSUE THREAD

**Issue by tbragin** [elastic/filebeat#128]
I ran through the following scenarios in testing Filebeat 1.0-beta4:
- [x] Filebeat 1.0-beta4 => ES 2.0 (no security)
- [x] Filebeat 1.0-beta4 => ES 1.7 (no security)
- [x] Filebeat 1.0-beta4 => LS 1.5.4 => ES 2.0 (no security)
- [x] Filebeat 1.0-beta4 => LS 1.5.4 => ES 1.7 (no security)
- [ ] Filebeat 1.0-beta4 => ES 2.0 + Shield 2.0 (with TLS)
- [x] Filebeat 1.0-beta4 => ES 1.7 + Shield 1.3 (with TLS)
- [ ] Filebeat 1.0-beta4 => LS 1.5.4 => ES 2.0 + Shield 2.0 (with TLS)
- [ ] Filebeat 1.0-beta4 => LS 1.5.4 => ES 1.7 + Shield 1.3 (with TLS)
- [ ] Filebeat 1.0-beta4 (with certs/SSL) => ES 2.0 (no security) 

Everything worked technically so far, however I hit a number of stumbling blocks and points of confusion on the documentation side, so here are my comments.

Getting started with Filebeat
*  [x] "Before starting filebeat, have a look at the configuration options from /etc/filebeat/filebeat.yml, described in the next chapter." Location could be somewhere else for the tar download for instance
*  [x] What’s a "prospector"? Term never defined. Could we say "locations" instead? I see that we have this in the code, but I don’t know if it makes sense to use this term in documentation?
*  [x] What’s a "harvester"? Term never defined. A thread, a process? I see that we have this in the code, but I don’t know if it makes sense to use this term in documentation?

Logstash configuration
*  [x] We should have instructions on how to install the beats input into Logstash (at least for the beta)
*  [x] Filebeat Getting Started section does not talk about how to connect Filebeat to Logstash; it should, or at least should link to a section walking through that setup. 
*  [x] In the Filebeat guide, we only describe Logstash output configuration in the migration guide from LSF, but new users will need to know about that setup as well. 
*  [x] There is no example configuration of the logstash-input-beats on the Logstash side in the Filebeat guide and no direct link to it in the Beats reference - it took me a while to locate this section https://www.elastic.co/guide/en/beats/libbeat/master/configuration.html#configuration-output
*  [x] Code coloring is off on this page https://www.elastic.co/guide/en/beats/filebeat/current/_logtash_forwarder_to_filebeat_migration.html

**Comment by tbragin** [comment]
cc: @dedemorton

**Comment by monicasarbu** [comment]
@ruflin Most of the issues are related to the docs. So, we can fix them even before GA, immediately after  we have a fix for them, right?

**Comment by ruflin** [comment]
@monicasarbu so far all problems are docs related. Concerning Harvester, Prospector we should perhaps readd the definition we had but in a better place. We should at least explain prospector as it also exists in the config file.

**Comment by ruflin** [comment]
@tbragin I made a checklist out of your feedback so we can mark it every time one of the issues above is resolved.

**Comment by dedemorton** [comment]
If we want to preserve the terminology "prospector" and "harvester", we should add a topic called "How Filebeat works" where we introduce those terms.

**Comment by ruflin** [comment]
@dedemorton Agree

**Comment by dedemorton** [comment]
Met with Tanya today to discuss some final improvements to the Getting Started topic for GA. I plan to create a short section in the Getting Started called something like Running Filebeat with Logstash. It'll basically just point back to the GS for libbeat, but this way users won't overlook the note and will definitely see that there is more to do to configure Logstash.

**Comment by dedemorton** [comment]
@tbragin Do you plan to test any of the other scenarios that are listed in your checklist. If not, we can close this issue because all the doc problems are fixed.

**Comment by monicasarbu** [comment]
Closing this issue as all the tasks were fixed. 
Thank you @dedemorton for taking care of the doc
Thank you @tbragin for the great feedback. Feel free to open another issue if you have other scenarios in mind.
